### PR TITLE
-added CoffeeBrush program

### PR DESCRIPTION
**Pull request by aniterstepanyan**

### CODE REVIEW

File: Coffee3.4/src/CoffeeBrush.java [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+            state = State.Off;
+        }
+
+
+    }
+    /*exit method*/
+    public boolean isOff (){
+        return state == State.Off;
+    }
+    public static void main(String args[]) {
```
Comment (hhov88) [inline Coffee3.4/src/CoffeeBrush.java]: Can you move logic to separate class, and create object on main method

Comment (aniterstepanyan) [reply Coffee3.4/src/CoffeeBrush.java]: Done :)

File: Coffee3.4/src/CoffeeBrush.java [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    private State state = State.Ready;
+
+    CoffeeBrush(){
+        setReady();
+    }
+
+    /**
+     * one cup of espresso contains:
+     * */
+    private int e_water = 250;
```
Comment (hhov88) [inline Coffee3.4/src/CoffeeBrush.java]: @aniterstepanyan always define fields in top of the class before other methods or constructor

File: Coffee3.4/src/CoffeeBrush.java [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+    private int e_coffee = 16;
+    private int e_cost = 4;
+
+    /**
+     * one cup of latte contains:
+     * */
+    private int l_water = 350;
```
Comment (hhov88) [inline Coffee3.4/src/CoffeeBrush.java]: @aniterstepanyan use "camelCase" for fields name
https://www.geeksforgeeks.org/java-naming-conventions/#:~:text=Java%20uses%20CamelCase%20as%20a,letter%2C%20rest%20all%20with%20capital.

File: Coffee3.4/src/CoffeeBrush.java [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+     * */
+    private int c_water = 200;
+    private int c_milk = 100;
+    private int c_coffee = 12;
+    private int c_cost = 6;
+
+    /**
+     * buy method
+     * */
+    private void buy(){
```
Comment (hhov88) [inline Coffee3.4/src/CoffeeBrush.java:57]: @aniterstepanyan this function is too big please separate it

File: Coffee3.4/src/CoffeeBrush.java [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+    /**
+     * take method
+     * */
+    private void take() {
+        System.out.println("I gave you $" + money);
+        money = money - money;
+    }
+
+    /**
+     * fill method
```
Comment (hhov88) [inline Coffee3.4/src/CoffeeBrush.java]: @aniterstepanyan try to write more meaning comments note like a "this is my code" or just method name

File: Coffee3.4/src/CoffeeBrush.java (hunk 2 above)
Comment (aniterstepanyan) [reply Coffee3.4/src/CoffeeBrush.java]: Done

File: Coffee3.4/src/CoffeeBrush.java (hunk 3 above)
Comment (aniterstepanyan) [reply Coffee3.4/src/CoffeeBrush.java]: Done

File: Coffee3.4/src/CoffeeBrush.java (hunk 5 above)
Comment (aniterstepanyan) [reply Coffee3.4/src/CoffeeBrush.java]: Done

**Review by hhov88 — APPROVED** [review]

**Pull request merged**